docs/source/conf.py

```diff
@@ -224,6 +225,27 @@ def adds(pth):
 nbsphinx_allow_errors = False  # False
 nbsphinx_timeout = 600  # seconds, set to -1 to disable timeout
 
+# since we don't execute the notebooks, we need to manually set the thumbnails here
```

Comment:
If you don't actually specify any thumbnails, you might as well not use `nbsphinx_thumbnails` at all. If no thumbnail is specified, the default one will be used.

See https://nbsphinx.readthedocs.io/en/0.8.6/gallery/no-thumbnail.html

And if you *do* want to specify a thumbnail for multiple examples, you can use wildcards, see https://github.com/spatialaudio/nbsphinx/blob/7e20521f44db47bfc72f4502d2782f205f0a3830/doc/conf.py#L29.

Reply:
Thanks @mgeier - I thought that specifying the thumbnails here would solve the issue with not executing the notebooks but turns out that the issue is something else as explained in https://github.com/spatialaudio/nbsphinx/issues/586